.docker/php8.0-apache/Dockerfile

```diff
@@ -33,16 +35,25 @@ ENV APACHE_DOCUMENT_ROOT /var/www/public
 # Install composer from image. You may change it to the latest
 COPY --from=composer:2.0 /usr/bin/composer /usr/bin/composer
 
+# Create system user to run Composer and Artisan Commands
+RUN useradd -G www-data,root -u $uid -d /home/$user $user
+RUN mkdir -p /home/$user/.composer && \
+    chown -R $user:$user /home/$user
+
 WORKDIR /var/www
 
 RUN sed -ri -e 's!/var/www/html!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/sites-available/*.conf
 RUN sed -ri -e 's!/var/www/!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/apache2.conf /etc/apache2/conf-available/*.conf
 
 # Some aliasses.
-RUN echo 'alias pa="php artisan"' >> ~/.bashrc
-RUN echo 'alias phpunit="./vendor/bin/phpunit"' >> ~/.bashrc
-RUN echo 'alias refresh="pa migrate:fresh"' >> ~/.bashrc
+RUN echo 'alias pa="php artisan"' >> /home/king/.bashrc
+RUN echo 'alias phpunit="./vendor/bin/phpunit"' >> /home/king/.bashrc
+RUN echo 'alias refresh="pa migrate:fresh"' >> /home/king/.bashrc
```

Comment:
I'd use `/home/$user/` here given you're already doing the same on lines [38-42] above

Reply:
```suggestion
RUN echo 'alias pa="php artisan"' >> /home/$user/.bashrc
RUN echo 'alias phpunit="./vendor/bin/phpunit"' >> /home/$user/.bashrc
RUN echo 'alias refresh="pa migrate:fresh"' >> /home/$user/.bashrc
```